### PR TITLE
enpitsu: docker: Tambah layanan Chibisafe

### LINKED ISSUE #48: Menambahkan provider chibisafe untuk file upload
https://github.com/chibisafe/chibisafe

Sudah dicoba, tinggal menambahkan provider ke docker compose

Ref: https://github.com/chibisafe/chibisafe?tab=readme-ov-file#docker

Ping @manoedinata

### PR DESCRIPTION
**Pull request by manoedinata**
Closes #48 

Disadur dari repositori Chibisafe [1]:

  Chibisafe is a file uploader service written in node that aims to to be
  easy to use and easy to set up. It's easy to use, easy to deploy, free
  and open source. It accepts files, photos, documents, anything you imagine
  and gives you back a shareable link for you to send to others.

Chibisafe akan digunakan sebagai file server untuk Enpitsu.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -41,6 +41,19 @@ services:
     expose:
       - 3000
 
+  chibisafe:
+    image: chibisafe/chibisafe:latest
+    restart: always
+    volumes:
+      - chibisafe_db:/home/node/chibisafe/database:rw
+      - chibisafe_uploads:/home/node/chibisafe/uploads:rw
+      - chibisafe_logs:/home/node/chibisafe/logs:rw
```
Comment (reacto11mecha) [inline docker-compose.yaml:50]: Ini nama usernya pasti `node` ya?

Comment (manoedinata) [reply docker-compose.yaml:50]: Iya, itu sudah bawaan dari image Chibisafe nya.

Comment (manoedinata) [reply docker-compose.yaml:50]: Yang perlu diubah hanya bagian volume host nya (`chibisafe_*`)

Comment (reacto11mecha) [reply docker-compose.yaml:50]: okey

Comment (reacto11mecha) [reply docker-compose.yaml:50]: Merge dulu aja dah, kalo emang salah nanti tinggal send fix lagi

**Pull request merged**